src/cli/utils/flags.go

```diff
@@ -215,6 +215,11 @@ func trimFolderSlash(folders []string) []string {
 	res := make([]string, 0, len(folders))
 
 	for _, p := range folders {
+		if len(p) == 1 && p[0] == path.PathSeparator {
```

Comment:
This is an overcomplicated form of equals:
```suggestion
		if p == path.PathSeparator {
```